ui/components/app/add-network/add-network.js

```diff
@@ -22,10 +29,13 @@ const AddNetwork = ({
   featuredRPCS,
 }) => {
   const t = useContext(I18nContext);
+  const theme = useSelector(getTheme);
```

Comment:
I guess this isn't working in storybook?
<img width="757" alt="Screen Shot 2022-04-13 at 11 13 37 AM" src="https://user-images.githubusercontent.com/8112138/163250365-5a6629f7-f930-47cc-a513-31df4028ac3a.png">
<img width="1440" alt="Screen Shot 2022-04-13 at 11 13 48 AM" src="https://user-images.githubusercontent.com/8112138/163250366-686ce8b1-759c-4ab8-adca-51a75b45c81a.png">

We could surface the prop in the component but I don't think that's necessary.

Reply:
@georgewrmarshall , you are correct. We can't get a redux state in the storybook. But it is working inside the application.

Reply:
@GuillaumeRx is there a way we could work in the storybook dark mode button so it changes the theme state in the provider wrapping storybook in `.storybook/preview.js`